pkg/sidecar/sidecar.go

```diff
@@ -66,16 +68,21 @@ func (s *Sidecar) CertReadyChan() <-chan struct{} {
 
 // updateCertificates Updates the certificates stored in disk and signal the Process to restart
 func (s *Sidecar) updateCertificates(svidResponse *workloadapi.X509Context) {
-	s.config.Log.Infof("Updating certificates")
+	s.config.Log.Info("Updating certificates")
 
-	if err := s.dumpBundles(svidResponse); err != nil {
-		s.config.Log.Errorf("Unable to dump bundle: %v", err)
+	err := s.dumpBundles(svidResponse)
+	if err != nil {
+		s.config.Log.WithError(err).Error("Unable to dump bundle")
 		return
 	}
 
-	if s.config.Cmd != "" {
+	if s.config.RenewSignal == "" {
+		s.config.Log.Warn("RenewSignal not loaded")
+	}
+
+	if s.config.Cmd != "" && s.config.RenewSignal != "" {
```

Comment:
it is  possible that cmd is provided but not signal and the same oposite, so this if will break things,
we have 3 possible scenaries here:
- cmd is set: we must run command alone
- signal is set: we must run signal alone
- cmd and signal are set: we must run both